docs/els-for-languages/php/README.md

```diff
@@ -150,6 +150,82 @@ Each version of PHP can be installed individually or all versions at once.
     { title: 'deb', content: 'apt-get install alt-php' }
   ]" />
 
+### Additional configurations (optional)
+
+The **default.ini** file is important for configuring alt-php. It sets default PHP settings and can be used to enable default extensions. We do not modify this file on our side. You need to update **default.ini** yourself to adjust PHP settings based on your Endless Lifecycle Support (ELS) usage and specific requirements.
+
+PHP extensions can be enabled or disabled in different configuration files. 
+* If you need to enable or disable an extension by *default* across all systems, list it by updating the `default.ini`.
+* If you want to enable or disable an extension only for a *specific* PHP version or setup, start by editing that extension's individual `.ini` file.
+
+#### Enabling a module through `default.ini`
+
+To enable or disable extensions in your installed PHP version:
+
+1. Open the `default.ini` file, usually located in the PHP configuration directory, in an editor of your choice: 
+
+<CodeWithCopy>
+
+```
+/opt/alt/phpXY/etc/php.d/default.ini
+```
+
+</CodeWithCopy>
+
+2. Edit the list of extensions:
+   * To enable an extension, remove the semicolon `;` at the beginning of the line.
+   * To disable an extension, add a semicolon `;` at the beginning of the line.
+   * If the extension line is missing, add it in this format: `extension=extension_name.so` (replace `extension_name.so` with the actual extension name).
+
+3. Save the changes in the `default.ini` file.
+
+#### Enabling a module through the configuration files
+
+Typically, extensions with their own `.ini` files are enabled or disabled directly in those files, allowing more control for a specific PHP configuration.
+* If you're unsure which file to modify to enable a specific extension, it's recommended to first try enabling it in the extension's own `.ini` file.
```

Comment:
If you want enable any extension by its own ini config, just copy needed ini file from /opt/alt/phpXY/etc/php.d.all/ to /opt/alt/phpXY/etc/php.d/
Be noticed: if the same extension presented in several ini configs which are copied to /opt/alt/phpXY/etc/php.d/, you get warnings in log file and maybe on your site.